config/hubs/pangeo-181919.cluster.yaml

```diff
@@ -0,0 +1,77 @@
+name: pangeo-181919
+provider: gcp
+gcp:
+  key: secrets/pangeo-181919.json
+  project: pangeo-181919
+  cluster: pangeo-hubs-cluster
+  zone: us-central1-b
+support:
+  config:
+    grafana:
+      ingress:
+        hosts:
+          - grafana.pangeo.2i2c.cloud
+        tls:
+          - secretName: grafana-tls
+            hosts:
+              - grafana.pangeo.2i2c.cloud
+hubs:
+  - name: coessing
+    domain: coessing.pangeo.2i2c.cloud
+    template: daskhub
+    auth0:
+      connection: google-oauth2
+    config:
+      basehub:
+        jupyterhub:
+          proxy:
+            service:
+              type: LoadBalancer
```

Comment:
Why we are using LoadBalancer here?
I saw that in AWS-based clusters (and it makes sense, AFAIK) but the only other GCP cluster where this is used is on meom cluster... wondering why that is not needed in other clusters such as the 2i2c or cloudbank ones?

Reply:
This is because this is a daskhub which sets a Cluster IP so Dask Gateway is accessed through JupyterHub: https://github.com/2i2c-org/pilot-hubs/blob/ec14dd8f13de016000cf679ad134648d20814cfb/hub-templates/daskhub/values.yaml#L200-L201

Without _also_ setting LoadBalancer, I was not getting an external IP from the proxy-public service - it was all just the cluster IP.

Reply:
OK, but that section you referenced belongs to a traefik config on the dask-gateway: https://github.com/2i2c-org/pilot-hubs/blob/master/hub-templates/daskhub/values.yaml#L97-L201 and not the proxy: https://github.com/2i2c-org/pilot-hubs/blob/master/hub-templates/basehub/values.yaml#L76, right? (or am I most likely missing something? :wink: )

> Without setting LoadBalancer, I was not getting an external IP from the proxy-public service - it was all just the cluster IP.

Ok, that explains why you set up the LoadBalancer in the proxy section, I still do not get why that is not needed in other clusters containing daskhubs.

Reply:
@yuvipanda and I ended up cribbing from the carbon plan cluster which does have this config https://github.com/2i2c-org/pilot-hubs/blob/e38f58c9c11e16928308bb9f9c401171e60fd3e1/config/hubs/carbonplan.cluster.yaml#L115-L117

Reply:
That's ok, my general question is about why this is needed on this pangeo (and meom) cluster but it is not needed in other GCP clusters. In any case, my question should not be a blocker here, and I will create a ticket to follow up with that discussion.